src/db/oracledb/jobs-dao.js

```diff
@@ -334,11 +337,21 @@ const createOrUpdateJob = async (operation, osuId, body) => {
   const connection = await getConnection('banner');
   try {
     let error;
-    const { studentEmployeeInd, changeReason: { code: changeReasonCode } } = body;
+    const { studentEmployeeInd, positionNumber, changeReason: { code: changeReasonCode } } = body;
     const employmentType = studentEmployeeInd ? 'student' : 'graduate';
 
-    if (employmentType === 'student' && !/^C5[0-2]/.test(body.positionNumber)) {
-      return new Error('Position number for students must start with C50, C51, or C52');
+    if (employmentType === 'student') {
+      const termination = ['TERME', 'TERMJ'].includes(changeReasonCode);
+      if (termination
+          && !validGradTermPositionNumberPrefixes.includes(positionNumber.substring(0, 3))) {
+        return new Error('Valid position numbers for termination must begin with one of these '
```

Comment:
I think `positionNumber.substring(0, 3)` can be stored in a variable and can be reused.